tests/projects/test_projects.py

```diff
@@ -12,32 +15,91 @@
 from mlflow.utils.file_utils import TempDir
 from mlflow.utils import env
 
-from tests.projects.utils import TEST_PROJECT_DIR, GIT_PROJECT_URI, validate_exit_status
-
+from tests.projects.utils import TEST_PROJECT_DIR, GIT_PROJECT_URI, TEST_DIR, validate_exit_status
+
+
+def _assert_dirs_equal(expected, actual):
+    dir_comparison = filecmp.dircmp(expected, actual)
+    assert len(dir_comparison.left_only) == 0
+    assert len(dir_comparison.right_only) == 0
+    assert len(dir_comparison.diff_files) == 0
+    assert len(dir_comparison.funny_files) == 0
+
+
+def test_fetch_project(tmpdir):
+    # Creating a local Git repo containing a MLproject file.
+    local_git = tmpdir.join('git_repo').strpath
```

Comment:
We don't actually currently test fetching this directory right? Did you want to add a test that uses this directory?